pythia/utils/general.py

```diff
@@ -52,18 +52,13 @@ def clip_gradients(model, i_iter, writer, config):
 
 
 def ckpt_name_from_core_args(config):
-    seed = config["training_parameters"]["seed"]
-
-    ckpt_name = "{}_{}_{}".format(
+    return "%s_%s_%s_%d" % (
```

Comment:
Can you remove changes in this function? Seed is randomized in current master so it is not possible to resume easily in slurm when you add seed to the ckpt name.

Reply:
Reverting the general.py to same as master. No changes in this file from my side anymore.